modal-js/examples/sandbox-tunnels.ts

```diff
@@ -0,0 +1,36 @@
+import { App } from "modal";
+
+const app = await App.lookup("libmodal-example", { createIfMissing: true });
+
+// Create a sandbox with Python's built-in HTTP server
+const image = await app.imageFromRegistry("python:3.12-alpine");
+const sandbox = await app.createSandbox(image, {
+  command: ["python3", "-m", "http.server", "8000"],
+  encryptedPorts: [8000],
+  timeout: 60000, // 1 minute
+});
+
+console.log("Sandbox created:", sandbox.sandboxId);
+
+console.log("Waiting for server to start...");
+await new Promise((resolve) => setTimeout(resolve, 3000));
+
+console.log("Getting tunnel information...");
+const tunnels = await sandbox.tunnels();
+const tunnel = tunnels[8000];
+
+console.log("Tunnel information:");
+console.log("  URL:", tunnel.url);
+console.log("  Port:", tunnel.port);
+
+console.log("\nMaking GET request to the tunneled server at " + tunnel.url);
+
+const response = await fetch(tunnel.url);
```

Comment:
Hmm, maybe you want to move the `setTimeout(..., 3000)` on line 16 to after line 19 in this example? The server may take time to start listening / bind to port after the tunnel is created (on sandbox startup).